chapter2_mdw/ExerciseFour

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/perl
+
+use 5.010;
+use warnings;
+
+chomp($numberone=<STDIN>);
+chomp($numbertwo=<STDIN>);
```

Comment:
- Good practice to probably add in some text to promp user? 
- Might want to consider adding some validation for the inputs as well. The script fails when non-numerics are entered.